### PR TITLE
master fix

### PR DESCRIPTION
**Pull request by SirNicholas**

### CODE REVIEW

File: features/step_definitions/article_steps.rb [hunk 1]
```diff
@@ -11,5 +11,5 @@
 ####################################
 
 Then /^I see comment displayed on (.*) page:$/ do |page, table|
-  expect(page.given.comment_data).to eql(table.rows_hash.symbolize_keys)
+  expect(page.given.comment_data).to eql(table.rows_hash.symbolize_keys).submit_form
```
Comment (romikoops) [inline features/step_definitions/article_steps.rb]: WTF? .submit_form

File: boot.rb [hunk 2]
```diff
@@ -4,6 +4,7 @@ def settings
   SexySettings::Base.instance()
 end
 
+require 'active_support/all'
```
Comment (romikoops) [inline boot.rb]: What the problem do we have without it?

File: features/step_definitions/article_steps.rb (hunk 1 above)
Comment (SirNicholas) [inline features/step_definitions/article_steps.rb]: Sorry, typo

File: boot.rb (hunk 2 above)
Comment (SirNicholas) [inline boot.rb]: ActiveSupport constant name not found

**Pull request merged**